dlrs/README.md

```diff
@@ -1,19 +1,21 @@
 ## Deep Learning Reference Stack (DLRS)
 
-### About
 
-The Deep Learning Reference Stack, is an integrated, highly-performant open source stack optimized for Intel® Xeon® Scalable platforms. 
+The Deep Learning Reference Stack, is an integrated, highly-performant open source stack optimized for Intel® Xeon® Scalable platforms.
 
 <img src="https://clearlinux.org/sites/default/files/single_2.png" width="400" height="300" />
 
+Highly-tuned and built for cloud native environments, the release enables developers to quickly prototype by reducing complexity associated with integrating multiple software components, while still giving users the flexibility to customize their solutions.
+
+The stack includes highly tuned software components across the operating system (Clear Linux OS), deep learning framework (TensorFlow*, PyTorch*), deep learning libraries (Intel® Math Kernel Library for Deep Neural Networks (MKL-DNN)) and other software components.
```

Comment:
Intel® Math Kernel Library for Deep Neural Networks -> oneAPI Deep Neural Network Library
MKL-DNN -> oneDNN